src/components/CommentInfo/CommentInfo.scss

```diff
@@ -1 +1,3 @@
-// add styles here
+.App__title {
+  text-align: center;
+}
```

Comment:
The `.App__title` class styling appears unrelated to the `CommentInfo` component. Consider moving this style to a more appropriate stylesheet, such as one associated with the `App` component, to maintain a clear separation of concerns.